### PR TITLE
Fixes issue #386

### PR DESCRIPTION
**Pull request by stephendonner**
Cleans up some of the iOS template a bit.

### CODE REVIEW

File: wikitemplate-ios.md [hunk 1]
```diff
@@ -27,78 +27,78 @@
...
+- [ ] Verify that you can add new favourites from the `Share with...` menu
 
 ## Context menus
 
 - [ ] Make sure context menu items in the URL bar work
 - [ ] Make sure context menu items on content work with no selected text
 - [ ] Make sure context menu items on content work with selected text
 - [ ] Make sure context menu items on content work inside an editable control (input, textarea, or contenteditable)
-- [ ] Context menu: verify you can Open in Background Tab, and Open in Private Tab
+- [ ] Context menu: verify you can `Open in New Tab` and `Open in Private Tab`
```
Comment (srirambv) [inline wikitemplate-ios.md:40]: This is not the same. Open tab in background is a different check from open in new tab. 

This check needs to be changes to `Verify you can links in a new tab or private tab. Ensure you can open links in background tabs.`

File: wikitemplate-ios.md [hunk 2]
```diff
@@ -108,47 +108,49 @@
...
 
 ## Content tests
 
-- [ ] Go to https://feedly.com/ and make sure that the password can be saved.  Make sure the saved password is auto-populated when you visit the site again
-- [ ] Open an email on http://mail.google.com/ or `inbox.google.com` and click on a link. Make sure it works
-- [ ] Verify that PDF is loaded over https at https://basicattentiontoken.org/BasicAttentionTokenWhitePaper-4.pdf
-- [ ] Verify that PDF is loaded over HTTP at http://www.pdf995.com/samples/pdf.pdf
-- [ ] Verify that https://mixed-script.badssl.com/ shows up as grey not red (no mixed content scripts are run)
-- [ ] Verify that search results from https://startpage.com/ open in a new tab (due to target being _blank_)
+- [ ] Go to `https://www.gmail.com` and make sure that the password can be saved.  Make sure the saved password is auto-populated when you visit the site again.
```
Comment (srirambv) [inline wikitemplate-ios.md:136]: Any reason to change this to Gmail? We had issues before where the prompt wasn't shown on certain devices. Prefer we keep it as is

**Review by srirambv — CHANGES_REQUESTED** [review]

File: wikitemplate-ios.md (hunk 1 above)
Comment (stephendonner) [reply wikitemplate-ios.md:40]: Thanks, will fix.

File: wikitemplate-ios.md (hunk 2 above)
Comment (stephendonner) [reply wikitemplate-ios.md:136]: Just "flow state/convenience" when testing, but that's not a great reason, especially due to the issue you mentioned above.  Will revert.

**Review by srirambv — APPROVED** [review]
++ thanks for fixing these

**Pull request merged**